spec/requests/students_spec.rb

```diff
@@ -1,7 +1,31 @@
 require 'rails_helper'
 
-RSpec.describe "Students", type: :request do
-  describe "GET /index" do
-    pending "add some examples (or delete) #{__FILE__}"
+RSpec.describe 'Students', type: :request do
+  describe 'GET /index' do
+    context 'when there are no students' do
+      it 'responds with 200 status' do
+        get '/students'
```

Comment:
🎱  I would recommend using Rails route helpers instead of writing out static strings. It's easier to update if path changes and less chance for things like spelling mistakes. You can find your route helpers by going to `localhost:3000/rails/info/routes`, or run `rails routes` in terminal

This one would most likely be: `students_path`

Reply:
Thanks! Was looking for route info earlier. Implemented.